source/agora/flash/Node.d

```diff
@@ -470,6 +484,37 @@ public abstract class FlashNode : ControlFlashAPI
             peer.gossipChannelsOpen(to_gossip);
     }
 
+    /// See `FlashAPI.gossipChannelUpdates`
+    public void gossipChannelUpdates (ChannelUpdate[] chan_updates)
+    {
+        writefln("%s: gossipChannelUpdates() with %s channels",
+            this.kp.V.flashPrettify, chan_updates.length);
+
+        ChannelUpdate[] to_gossip;
+        foreach (update; chan_updates)
+        {
+            if (auto conf = update.chan_id in this.known_channels)
+            {
+                auto pk = update.direction == PaymentDirection.TowardsPeer ?
+                                                conf.funder_pk : conf.peer_pk;
+                if (auto chan_update = update.chan_id in this.channel_updates)
+                    if (auto dir_update = update.direction in *chan_update)
+                        if (*dir_update == update)
+                            continue;
```

Comment:
I was hoping someone could show me a better way of doing this:D

Reply:
Possibly via `getDefault`? Let me check it when I get in the office.